core/dbt/logger.py

```diff
@@ -32,6 +32,7 @@
 )
 
 SECRET_ENV_PREFIX = "DBT_ENV_SECRET_"
+DEFAULT_ENV_PLACEHOLDER = "DBT_DEFAULT_PLACEHOLDER"
```

Comment:
When it was just the SECRET_ENV_PREFIX, it felt kind of okay to have it in the logger file. But I'm not sure it does now. Can you think of a better place to put them? Particularly since eventually we do intend to get rid of logger.py...